### PR TITLE
LPD-17467 Importing Object - Custom State Flow doesn't resemble JSON file

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPD-17467](https://liferay.atlassian.net/browse/LPD-17467)

### CODE REVIEW

**Review by linolaoj — CHANGES_REQUESTED** [review]
Hi @diogomelo09, can we change the order of the if clause here to make it easier to read? (I mean avoiding the negative !)

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java [hunk 1]
```diff
@@ -89,11 +89,20 @@ public class ObjectFieldSettingUtil {
 		if (serviceBuilderObjectFieldSetting.compareName(
 				ObjectFieldSettingConstants.NAME_STATE_FLOW)) {
 
+			ObjectStateFlow objectStateFlow = null;
+
+			if (!(objectFieldSetting.getValue() instanceof ObjectStateFlow)) {
```
[the remark this answers is not in the document]
Comment (diogomelo09) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java]: Done!! :white_check_mark:

**Comment by linolaoj** [comment]
ci:test:object

**Comment by linolaoj** [comment]
Hi @diogomelo09 can you please check the test PublishAPIApplication#CanGetPublishedStatusInResponse. If it is not related then you can forward the pr.

**Comment by rijema** [comment]
New PR associated with the JIRA ticket.

**Pull request closed without merging**